hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/RejectReplicationRequestStateChecker.java

```diff
@@ -1,4 +1,6 @@
-/**
+/*
+ * Copyright The Apache Software Foundation
```

Comment:
What's this?

Reply:
We've had this discussion before. Its not javadoc so seems right to remove the extra '*'. We should do up an agreement on way to go. Did you say why you think we should preserve the '**'? Is it because its in the formatter?

Reply:
I mean the 'Copyright The Apache Software Foundation'.